modules/log6x-logging-compatibility-support-matrix.adoc

```diff
@@ -0,0 +1,24 @@
+[id="log6x-logging-compatibility-support-matrix_{context}"]
+= Compatibility and support matrix
+
+Some features in this release are currently in link:https://access.redhat.com/support/offerings/techpreview[Technology Preview]. These experimental features are not intended for production use.
+
+In the table, features are marked with the following statuses:
+
+[horizontal]
+TP:: Technology Preview
+GA:: General Availability
+
+.Compatibility and support matrix
+[options="header"]
+|===
+
+| {clo} Version 6+| Component Version | OpenShift Version | Support Status
+
+| Operator | `eventrouter` | `logfilemetricexplorer` | `loki` | `lokistack-gateway` | `opa-openshift` | `vector` | |
+
+|6.0 | 0.4 (GA or TP) | v1.1 (GA or TP) | v2.9.4 (GA or TP) | v0.1 (GA or TP) | v0.1 (GA or TP) | v0.37.x (GA or TP) | 4.16 | GA or TP
```

Comment:
TP or GA here is N/A and unnecessary. None of the constituents components of logging are TP, only forwarding to OTEL.

Reply:
It is good to mention the TP/GA status for components/features even if we do not have any component that is in TP state right now. In future, we might have such TP components. This would be useful information for our users.

Reply:
I disagree. We have never released the underlying component version before, as the they cannot be changed by the user.  I think it will be confusing and raise questions about prior versions which are not relevant. If in the future there are TP components, they can state as much in the version fields, this does not need its own column. As a column, it does not work given the format of the table. What would it apply to?

Reply:
Got your point, Libby. I will remove that separate column for Support status from the table.

Reply:
This column was not removed prior to merge.